server/server.go

```diff
@@ -875,6 +887,33 @@ func (s *server) getClusterInfo() *rpcpb.ClusterInfo {
 	return info
 }
 
+func (s *server) handleUnexpectedNodeStop(unexpectedNodeStopCh chan network.UnexpectedNodeStopMsg) {
+	for {
+		unexpectedStopMsg := <-unexpectedNodeStopCh
```

Comment:
Isn't this possibly blocking forever?

Reply:
here, only the ANR server will read the channel